src/js/i18n/locales/ar.json

```diff
@@ -34,5 +34,6 @@
   "Home.Secondarycontent.bodytext": "Alpha.austin.gov هو موقع إلكتروني جديد والعمل قيد التنفيذ. للحصول على كامل موقع مدينة أوستن، قم بزيارة {citySiteLink}. تعرف على المزيد حول الموقع الإلكتروني الجديد عبر {projectsSiteLink}.",
   "Service.RelatedLinks.SectionHeader": "الاطلاع على الخدمات ذات الصلة",
   "Service.RelatedLinks.Tag": "الخدمة",
-  "Home.RelatedLinks.Tag": "الخدمة"
+  "Home.RelatedLinks.Tag": "الخدمة",
+  "Submenu.workInProgress": "Alpha.austin.gov هو موقع إلكتروني جديد والعمل قيد التنفيذ. للحصول على كامل موقع مدينة أوستن، قم بزيارة {citySiteLink}. تعرف على المزيد حول الموقع الإلكتروني الجديد عبر {citySiteLink}."
```

Comment:
Same here -- should the last link be {projectSiteLink}? This text looks a lot like line 34 "Home.Secondarycontent.bodytext" but the last link is different

Reply:
same as above